Example/BitmovinConvivaAnalytics/ViewController.swift

```diff
@@ -31,12 +31,12 @@ class ViewController: UIViewController {
 
         setupBitmovinPlayer()
 
-        if let posterUrl = sourceItem.posterSource {
+        /*if let posterUrl = sourceItem.posterSource {
             // Be aware that this will be executed synchronously on the main thread (change to SDWebImage if needed)
             if let data = try? Data(contentsOf: posterUrl) {
                 posterImageView.image = UIImage(data: data)
             }
-        }
+        }*/
```

Comment:
Why commenting this out? 🤔 Please either remove it or keep it 🙂

Reply:
Uncommented the code.